agent/apiharness/src/main/java/com/intuit/tank/harness/APITestHarness.java

```diff
@@ -661,11 +661,11 @@ public String getInstanceId() {
     }
 
     public long getSimulationEndTimeMillis() {
-        return getStartTime() + agentRunData.getSimulationTime();
+        return getStartTime() + agentRunData.getSimulationTimeMillis();
     }
 
     public boolean hasMetSimulationTime() {
-        if (agentRunData.getSimulationTime() > 0) {
+        if (agentRunData.getSimulationTimeMillis() >= 0) {
```

Comment:
If simulation time defaults or is set to 0, run() in TestPlanStarter will run infinitely

Reply:
If a test is started with parameter `Run Scripts Until`: `Scripts Loops Completed`, What is the expected value of this?

Reply:
Resolved offline